src/compiler/generate-common-tests.xsl

```diff
@@ -28,8 +28,8 @@
       select="xs:anyURI('http://www.jenitennison.com/xslt/xspec')" />
 
    <!-- XSpec namespace prefix -->
-   <xsl:variable name="xspec-prefix" as="xs:string">
-      <xsl:variable name="e" select="/element()" as="element(x:description)" />
+   <xsl:function name="x:xspec-prefix" as="xs:string">
+     <xsl:param name="e" as="element()" />
```

Comment:
```suggestion
      <xsl:param name="e" as="element()" />
```

Reply:
Fixed in 21ed0e1f844324b13670d7ad7828bb9894d2a26f